README.md

```diff
@@ -1 +1,56 @@
-# scm-router
\ No newline at end of file
+# SCM Router
+[![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][status-image]][status-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
+
+> A generic scm plugin that routes builds to a specified scm
+
+A scm is interface for interacting with a source control management system.
+
+i.e. Github, Bitbucket, Gitlab
+
+The scm router will allow multiple scms to be used in a Screwdriver cluster.
+
+## Usage
+
+```bash
+npm install screwdriver-scm-router
+```
+
+### Interface
+
+It will initialize any routers specified in the [default.yaml](https://github.com/screwdriver-cd/screwdriver/blob/master/config/default.yaml#L123-L156) under the `scm` and `scms` keyword. At least one of `scm` and `scms` is necessarily required.
```

Comment:
At least one is required. It is okay to set both. Even if only one setting is okay.